### PR TITLE
Autocomplete demo

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-autocomplete-demo](https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-autocomplete-demo)

File: src/app/public/plugin-resources/code-examples/autocomplete/advanced/autocomplete-demo.component.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  FormGroup
+} from '@angular/forms';
+
+import {
+  SkyAutocompleteSearchFunctionFilter,
+  SkyAutocompleteSelectionChange
+} from '@skyux/lookup';
+
+@Component({
+  selector: 'app-autocomplete-demo',
```
Comment (blackbaud-johnly) [inline src/app/public/plugin-resources/code-examples/autocomplete/advanced/autocomplete-demo.component.ts:17]: Is it OK that each of the code samples uses the same selector? I assume so, but just making sure.

**Review by blackbaud-johnly — APPROVED** [review]
LGTM. I added one question, but I'm pretty sure it's nothing to worry about. ... I went through the code samples to check how you split them out and tested them in Stackblitz, but it wouldn't be a bad idea to get a developer to check the code samples too.

File: src/app/public/plugin-resources/code-examples/autocomplete/advanced/autocomplete-demo.component.ts (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/plugin-resources/code-examples/autocomplete/advanced/autocomplete-demo.component.ts:17]: Yep, they run in isolation, so there's no conflict.

**Review by Blackbaud-DeniseCisneros — APPROVED** [review]

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**